live-site/components/event-schedule-section/event-item/EventItem.styles.tsx

```diff
@@ -0,0 +1,211 @@
+import styled from 'styled-components';
+import { max } from '../../../../shared-ui/lib/responsive';
+import { colors } from '../../../../shared-ui/style/colors';
+import { H3, P } from '../../../../shared-ui/style/typography';
+
+const StyledEventItemMobileContainer = styled.div`
+  @media ${max.tablet} {
+    display: flex;
+  } ;
+`;
+const StyledEventItemContainer = styled.div`
+  margin-left: 1em;
+  margin-right: 2em;
+  margin-top: 1em;
+  border-radius: 2em;
+  height: 8em;
+  background-color: ${colors.WHITE};
+`;
+
+const StyledEventItemLeftSideContainer = styled.div`
+  display: flex;
+  float: left;
+
+  @media ${max.tabletLg} {
+    padding-left: 2em;
+    display: flex;
+  } ;
+
+  @media ${max.tablet} {
+    display: block;
+  } ;
+
+  @media ${max.tabletXs} {
+    display: block;
+    padding-left: 0.5em;
+  } ;
+`;
+
+
+
+const StyledEventItemNameLocationContainer = styled.div`
+  display: block;
+
+  @media ${max.tabletLg} {
+    padding-top: 0em;
+    
+    
+  }
+
+  @media ${max.tablet} {
+    padding-top: 0;
+  }
+
+  @media ${max.mobile} {
+    padding-top: 0;
+    padding-left: 1em;
+  } ;
+`;
+
+const StyledPinpointLocationContainer = styled.div`
+  display: flex;
+`;
+
+const StyledEventTimeTopicContainer = styled.div`
+  padding: 2em;
+
+  @media ${max.tabletLg} {
+    padding-left: 0em;
+  }
+
+  @media ${max.tablet} {
+    display: flex;
+    padding-left: 1em;
+    padding-right: 0;
+    padding-bottom: 0;
+  }
+
+  @media ${max.tabletXs} {
+    padding-left: 1em;
+  }
+
+  @media ${max.mobile} {
+    display: flex;
+    padding-left: 1em;
```

Comment:
remove padding-left: 1em; b/c we want these two to be aligned
![Uploading Screenshot 2023-01-03 at 2.23.41 PM.png…]()